src/nodedef.cpp

```diff
@@ -1437,6 +1593,43 @@ void CNodeDefManager::resetNodeResolveState()
 	m_pending_resolve_callbacks.clear();
 }
 
+void CNodeDefManager::mapNodeboxConnections()
+{
+	for (u32 i = 0; i < m_content_features.size(); i++) {
+		ContentFeatures *f = &m_content_features[i];
+		if ((f->drawtype != NDT_NODEBOX) && (f->node_box.type != NODEBOX_CONNECTED))
```

Comment:
Yup, since the default value is an unused type, this probably didn't break anything, but it's indeed wrong.